### DISCUSSION TITLE
oltacidergisi.com - popup

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uAssets#23094] [Q&A] [answered]
It appears on every page refresh. I don't know why they show this popup.
Can you please suggest a user filter to hide this popup?

```
https://oltacidergisi.com/haberler/hayalet-balikcilik-raporunda-dikkat-ceken-tespitler/
```

<details open>
<summary>screen</summary>

![resim](https://github.com/uBlockOrigin/uAssets/assets/165385945/d6b6e327-854e-443c-b70f-71fd417dce3a)

</details>

**Comment by gorhill** [comment, marked as answer]
This works:

    oltacidergisi.com##w3m-modal
    oltacidergisi.com##html:style(overflow:auto!important)

> I don't know why they show this popup.

Maybe that site has been compromised?

**Reply by stephenhawk8054** [reply]
Looks like a known malware affects the site: https://blog.sucuri.net/2024/02/web3-crypto-malware-angel-drainer.html

I did cover it partially few weeks ago: https://github.com/uBlockOrigin/uAssets/commit/2f6147042af63ddcbf47fa12427146cef78621f6

**Comment by ghost** [comment]
@gorhill 
It works. Thanks for help.

**Comment by gorhill** [comment]
Better: This defuse the injected malware code:

    oltacidergisi.com##+js(acis, atob, minimalDrainValue)

No need for the two other filters with this.
